File: Java/src/br.com.geekuniversity.com.section02/S02e02.java
package br.com.geekuniversity.com.section02;

import java.util.Scanner;

public class S02e02 {
    public static void main(String[] args) {
        float numberOne, numberTwo, result;

        Scanner userInput =  new Scanner(System.in);
        
        System.out.println("Please enter a number for number one: ");
        numberOne = userInput.nextFloat();
        
        System.out.println("Please enter a number for number two: ");
        numberTwo = userInput.nextFloat();

        result = (numberOne + numberTwo) * numberOne;

        System.out.println("\nThe result is: " + result);
        
        userInput.close();

    }
}

File: C/section02/S02E02.c
#include <stdio.h>

int main() {
    int numberOne, numberTwo, result;

    printf("Please type the number one:\n");
    scanf("%d", &numberOne);
    
    printf("Please type the number two:\n");
    scanf("%d", &numberTwo);

    result = (numberOne + numberTwo) * numberOne;

    printf("\nThe result is %d\n", result);

    return 0;
}